components/FileUploader.js

```diff
@@ -81,18 +86,27 @@ class FileUploaderButton extends Component {
 
 class Filename extends Component {
   static propTypes = {
-    name: PropTypes.string,
     style: PropTypes.object,
     status: PropTypes.oneOf(['edit', 'complete', 'uploading']),
+    editIconDescrition: PropTypes.string,
+    completeIconDescrition: PropTypes.string
   };
 
   static defaultProps = {
+    completeIconDescrition: 'File has been uploaded',
+    editIconDescrition: 'Remove file',
```

Comment:
so maybe I'm misreading this but I don't think these can actually be overriden since these props can't be set on the parent component and passed down.